src/osd/PG.cc

```diff
@@ -4108,7 +4108,7 @@ void PG::chunky_scrub(ThreadPool::TPHandle &handle)
             }
 
             // reset handle once in a while, the search maybe takes long.
-            if (++loop >= g_conf->osd_loop_before_reset_tphandle) {
+            if (++loop >= g_conf->osd_scan_list_ping_tp_interval) {
               handle.reset_tp_timeout();
```

Comment:
osd_scan_list_ping_tp_interval is about IO.  I would actually prefer to have two configs here.  Please remove this commit.